.azure/pipelines/ci.yml

```diff
@@ -675,18 +676,15 @@ stages:
         dependsOn:
           - Windows_build
           - Windows_arm_build
+          - Windows_64_build
```

Comment:
/fyi @JunTaoLuo as I mentioned in the description, this was not a big deal but is something to keep in mind if / when we add more build jobs e.g. if we split the Linux_x64 build up

Reply:
Why did we name that job `Windows_64`? Shouldn't it be `Windows_Arm64`?

Reply:
Fixed that @wtgodbe